### ISSUE TITLE
getting error "Failed to save settings. Please restart IntelliJ IDEA"

### ISSUE THREAD

**Issue by JurgenCruz** [JavaSaBr/IntelliJ-jME-Plugin#4]
I have the latest version of the plugin and getting this exception. I downloaded the project to see if intellij recognizes the assets folder as resources but that didn't work. i had an existing project so no sure if that matters. Also I don't have the jmonkeybuilder. is that an issue? Can I use this plugin without jmonkeybuilder? should I uninstall this plugin if it is only for jmonkey builder?

Thanks.

```
2018-05-01 11:37:52,511 [  18822]   WARN - mponents.impl.stores.StoreUtil - Save settings failed 
java.lang.RuntimeException: java.lang.Exception: Cannot get JmePluginComponent component state
	at com.intellij.util.ExceptionUtil.rethrow(ExceptionUtil.java:174)
	at com.intellij.util.lang.CompoundRuntimeException.throwIfNotEmpty(CompoundRuntimeException.java:153)
	at com.intellij.configurationStore.ComponentStoreImpl.save(ComponentStoreImpl.kt:174)
	at com.intellij.openapi.components.impl.stores.StoreUtil.save(StoreUtil.java:40)
	at com.intellij.openapi.application.impl.ApplicationImpl.saveSettings(ApplicationImpl.java:1414)
	at com.intellij.openapi.components.impl.stores.StoreUtil.saveDocumentsAndProjectsAndApp(StoreUtil.java:111)
	at com.intellij.ide.actions.SaveAllAction.actionPerformed(SaveAllAction.java:23)
	at com.intellij.openapi.actionSystem.ex.ActionUtil$1.run(ActionUtil.java:220)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:237)
	at com.intellij.openapi.actionSystem.impl.ActionButton.actionPerformed(ActionButton.java:193)
	at com.intellij.openapi.actionSystem.impl.ActionButton.performAction(ActionButton.java:153)
	at com.intellij.openapi.actionSystem.impl.ActionButton.processMouseEvent(ActionButton.java:357)
	at java.awt.Component.processEvent(Component.java:6306)
	at java.awt.Container.processEvent(Container.java:2237)
	at java.awt.Component.dispatchEventImpl(Component.java:4897)
	at java.awt.Container.dispatchEventImpl(Container.java:2295)
	at java.awt.Component.dispatchEvent(Component.java:4719)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4889)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4526)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4467)
	at java.awt.Container.dispatchEventImpl(Container.java:2281)
	at java.awt.Window.dispatchEventImpl(Window.java:2746)
	at java.awt.Component.dispatchEvent(Component.java:4719)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:764)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:90)
	at java.awt.EventQueue$4.run(EventQueue.java:737)
	at java.awt.EventQueue$4.run(EventQueue.java:735)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:734)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:837)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:762)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:434)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Caused by: java.lang.Exception: Cannot get JmePluginComponent component state
	at com.intellij.configurationStore.ComponentStoreImpl.doSaveComponents(ComponentStoreImpl.kt:222)
	at com.intellij.configurationStore.ComponentStoreImpl.save(ComponentStoreImpl.kt:154)
	... 41 more
Caused by: java.lang.IllegalArgumentException: Extension is missing for storage file: C:/Users/Jurgen/.IdeaIC2018.1/config/options/JmePluginComponent
	at com.intellij.configurationStore.StateStorageManagerImpl.createStateStorage(StateStorageManagerImpl.kt:270)
	at com.intellij.configurationStore.StateStorageManagerImpl.getOrCreateStorage(StateStorageManagerImpl.kt:187)
	at com.intellij.configurationStore.StateStorageManagerImpl.getOrCreateStorage$default(StateStorageManagerImpl.kt:171)
	at com.intellij.configurationStore.StateStorageManagerImpl.getStateStorage(StateStorageManagerImpl.kt:149)
	at com.intellij.configurationStore.StateStorageManagerImpl$startExternalization$1.setState(StateStorageManagerImpl.kt:430)
	at com.intellij.configurationStore.ComponentStoreImpl.commitComponent(ComponentStoreImpl.kt:269)
	at com.intellij.configurationStore.ComponentStoreImpl.doSaveComponents(ComponentStoreImpl.kt:215)
```

**Comment by JavaSaBr** [comment]
I will look at it.

**Comment by JavaSaBr** [comment]
Fixed in the version 1.3.4
